nuxeo-web-ui-ftest/test/pages/ui/browser/document_form_layout.js

```diff
@@ -3,17 +3,24 @@
 import BasePage from '../../base';
 import DocumentLayout from './document_layout';
 
-export default class DocumentEdit extends BasePage {
-  constructor(selector, docType) {
+export default class DocumentFormLayout extends BasePage {
+  constructor(selector, docType, layout) {
     super(selector);
     this._docType = docType;
+    this._layout = layout;
   }
 
   set title(title) {
     return this.el.element('.input-element input').setValue(title);
   }
 
-  layout() {
+  get layout() {
     return new DocumentLayout(`nuxeo-${this._docType.toLowerCase()}-edit-layout`);
```

Comment:
We should be using `this._layout` here instead of an hardcoded "edit":
```
return new DocumentLayout(`nuxeo-${this._docType.toLowerCase()}-${this._layout}-layout`);
```

Reply:
Thx 👍 (fyi there's a new feature in GH review comments to suggest editions perhaps we should take it for a spin ;))